### PR TITLE
feat: use 'Github Actions' to run tests and auto release, fix #5

### LINKED ISSUE #5: Add GitHub Actions CI and code coverage, to fix the badges

### PR DESCRIPTION
**Pull request by basakest**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@techoner @gaobinzhan please review.

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,12 @@
         "ext-swoole": ">=4.5",
         "casbin/casbin": "^2.2",
         "easyswoole/orm": "^1.4",
-        "easyswoole/easyswoole": "3.x"
+        "easyswoole/easyswoole": "~3.1|~3.2"
```
Comment (Player626) [inline composer.json]: The frame version is 3.4.x.

Comment (basakest) [reply composer.json]: Oh, yes. It should be ~3.4|~3.3. I accidentally spelled it wrong.

**Pull request merged**